packages/xo-web/src/xo-app/backup-ng/new/metadata/index.js

```diff
@@ -265,6 +290,11 @@ export default decorate([
       missingSchedules,
     } = state.showErrors ? state : {}
 
+    const { reportWhen = 'failure' } = defined(
```

Comment:
Do you want to add `reportWhen` to the Overview default values filter so it doesn't show up in the "Notes" when it's set to `'failure'`?